lib/model/query/entities.js

```diff
@@ -23,8 +23,8 @@ const { runSequentially } = require('../../util/promise');
 // ENTITY CREATE
 
 const _defInsert = (id, root, creatorId, userAgent, label, json, version, sourceId = null) => sql`
-  insert into entity_defs ("entityId", "root", "sourceId", "creatorId", "userAgent", "label", "data", "current", "createdAt", "version")
-  values (${id}, ${root}, ${sourceId}, ${creatorId}, ${userAgent}, ${label}, ${json}, true, clock_timestamp(), ${version})
+  insert into entity_defs ("entityId", "root", "sourceId", "creatorId", "userAgent", "label", "data", "current", "createdAt", "version", "baseVersion", "dataReceived")
+  values (${id}, ${root}, ${sourceId}, ${creatorId}, ${userAgent}, ${label}, ${json}, true, clock_timestamp(), ${version}, ${version - 1}, ${json})
```

Comment:
will this insert baseVersion=0 for new entities? it could use root (or version, but root seems more explicit) to conditionally set baseVersion to null for new entities.